packages/liveblocks-client/src/room.ts

```diff
@@ -892,7 +893,13 @@ function makeStateMachine<
     }
   }
 
+  function isReadonly(scopes: string[]) {
+    return scopes.includes("room:read");
```

Comment:
Here is a more philosophical question: should we consider a connection read _only_ only if they also don’t have write permissions?

If I have scopes `['room:write', 'room:read']` then I would not think of that as a readonly connection?

Reply:
Perhaps?

```suggestion
    return scopes.includes("room:read") && !scopes.includes("room:write");
```